### PR TITLE
Added overview figure in SVG and PNG

### PR DESCRIPTION
**Pull request by x-tabdeveloping**
I added the overview figure for the paper.
The SVG file was created with Inkscape, and is editable in it. I have no idea why when rendered in the browser, some text is missing, but it shows up in Inkscape and if you render the image it looks as it should.

### CODE REVIEW

**Comment by x-tabdeveloping** [comment]
Ahhh it looks whack in the paper, I will try to make it wider

File: docs/images/mteb_overview.svg
[the remark this answers is not in the document]
Comment (x-tabdeveloping) [reply docs/images/mteb_overview.svg:1]: Don't know why, but it seems missing when rendered in the browser. It's visible in Inkscape, and when rendered from it, it looks fine

[the remark this answers is not in the document]
Comment (KennethEnevoldsen) [reply docs/images/mteb_overview.svg:1]: as long as it is there in the downloaded version it is fine

**Comment by KennethEnevoldsen** [comment]
#940 fixes tests

**Pull request merged**